services/QuillLMS/app/services/google_integration/classroom_students_data.rb

```diff
@@ -0,0 +1,30 @@
+module GoogleIntegration
+  class ClassroomStudentsData
+    include Enumerable
+
+    attr_reader :classroom, :classroom_students_client
+
+    delegate :google_classroom_id, to: :classroom
```

Comment:
Never seen this before! Cool!

Reply:
Thanks.  Sometimes if you're dealing with an object that is effectively a collection you can use Enumerable to get methods like `filter`, `select`, etc.